app/Models/Foundation/Summit/Events/Presentations/Actions/AllowedPresentationActionType.php

```diff
@@ -0,0 +1,119 @@
+<?php namespace models\summit;
+/**
+ * Copyright 2022 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+use App\Models\Foundation\Main\IOrderable;
+use App\Models\Foundation\Summit\SelectionPlan;
+use models\utils\One2ManyPropertyTrait;
+use models\utils\SilverstripeBaseModel;
+use Doctrine\ORM\Mapping AS ORM;
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="PresentationActionType_SelectionPlan")
+ * Class AllowedPresentationActionType
+ * @package models\summit
+ */
+class AllowedPresentationActionType extends SilverstripeBaseModel implements IOrderable
```

Comment:
inherit from BaseEntity ( SSBase model provide classType, and timestamp that does not needed on a join table)

Reply:
done